nbodykit/core/painter/DefaultPainter.py

```diff
@@ -68,19 +69,19 @@ def paint(self, pm, datasource):
                         if not self.interlaced:
                             self.basepaint(real, position, paintbrush=self.paintbrush)
                         else:
-                            self.interlaced_paint(real, real2, position, paintbrush=self.paintbrush)
+                            self.shiftedpaint(real, real2, position, paintbrush=self.paintbrush)
                         Nlocal += len(position)
                 else:
                     for position, weight in stream.read(['Position', self.weight]):
                         if not self.interlaced:
                             self.basepaint(real, position, weight=weight, paintbrush=self.paintbrush)
                         else:
-                            self.interlaced_paint(real, real2, position, weight=weight, paintbrush=self.paintbrush)
+                            self.shiftedpaint(real, real2, position, weight=weight, paintbrush=self.paintbrush)
                         Nlocal += len(position)
             c1 = real.r2c()
             c2 = real2.r2c()
-            H = self.pm.BoxSize / self.pm.Nmesh
 
+            H = pm.BoxSize / pm.Nmesh
```

Comment:
This is a bit suspicious to me. @nickhand does this look right?
It appears to be
```
s1 += s2 * 0.5 * exp(I * k * (L / 2N))
```